src/ui/units/dash/store/utils.ts

```diff
@@ -0,0 +1,51 @@
+import {generateUniqId} from '@gravity-ui/dashkit';
+import {I18n} from 'i18n';
+import {DL} from 'ui/constants';
+import Utils from 'ui/utils';
+
+import {Mode} from '../modules/constants';
+
+const storeI18n = I18n.keyset('dash.store.view');
+const dashCreateI18n = I18n.keyset('component.dialog-create-dashboard.view');
+
+export const getFakeDashEntry = (workbookId?: string) => {
+    const salt = Math.random().toString();
+    const {counter, id: newTabId} = generateUniqId({salt, counter: 0, ids: []});
+    const initialKey = `${DL.USER_FOLDER}${dashCreateI18n('label_default-name')}`;
```

Comment:
Using the same i18n key as the initial name in the dashboard creation modal (`DialogCreateDashboard`)